docs/user/custom-resources/apirule/04-10-apirule-custom-resource.md

```diff
@@ -42,6 +42,7 @@ This table lists all parameters of APIRule CRD together with their descriptions:
 > [!WARNING]
 >  Having both the Oathkeeper and Istio `jwt` access strategies defined is not supported. Access
 > strategies `noop`, `allow`, or `no_auth` cannot be used with any other access strategy on the same **spec.rules.path**.
+> Mixing secure and unsecure access strategies is not supported.
```

Comment:
```suggestion
> You must not use the access strategies `jwt`, `oath2_introspection`, `noop`, `allow`, or `no_auth` with any other access strategy on the same **spec.rules.path**.
> Having both the Oathkeeper and Istio `jwt` access strategies defined is not supported.
```
By 'secure access strategies' we mean `jwt` and `oath2_introspection`, right? I assume that there still are cases when the user can define more than two access strategies